modules/textlocalwrapper.py

```diff
@@ -0,0 +1,36 @@
+import json
+from six.moves.urllib import request, parse
+from cshsms.settings import TEXTLOCAL_API, TEXTLOCAL_PRIMARY_ID
+
+class TextLocal(object):
+	def __init__(self, apikey, primary_id):
+		self.apikey = apikey
+		self.primary_id = primary_id
+
+	def get_all_inboxes(self):
+		params = {'apikey': self.apikey}
+		inboxes_url = 'https://api.textlocal.in/get_inboxes/?'
+		f = request.urlopen(inboxes_url + parse.urlencode(params))
+		return json.loads(f.read().decode('latin1')), f.code
+
+	def get_primary_inbox(self):
+		params = {'apikey': self.apikey, 'inbox_id': self.primary_id}
+		messages_url = 'https://api.textlocal.in/get_messages/?'
+		f = request.urlopen(messages_url + parse.urlencode(params))
+		return json.loads(f.read().decode('latin1')), f.code
```

Comment:
Yeah, on reflection I'd probably say I should make 13-14 and 19-20 into it's own method. Will do.

And no, `f.code` is never actually used and can be changed to `_`, or the prior line changed to only get the needed data